src/Monticello/MCSubDirectoryRepository.class.st

```diff
@@ -75,11 +75,10 @@ MCSubDirectoryRepository >> loadAllFileNames [
 
 { #category : #'i/o' }
 MCSubDirectoryRepository >> readStreamForFileNamed: aString do: aBlock [
-	| file val |
-	file := FileStream readOnlyFileNamed: (self findFullNameForReading: aString).
-	val := aBlock value: file.
-	file close.
-	^ val
+
+	| fileReference |
+	fileReference := (self findFullNameForReading: aString) reference.
```

Comment:
sure about that #reference send ?

Reply:
Yes, `findFullNameForReading:` returns a `FileSystemDirectoryEntry`.